configure.ac

```diff
@@ -1,4 +1,4 @@
-AC_INIT([libzbxpgsql], [1.1.0], [ryan@cavaliercoder.com])
+AC_INIT([libzbxpgsql], [1.1.1], [ryan@cavaliercoder.com])
```

Comment:
I'll update it.

On Sat, Aug 27, 2016 at 10:38 PM, Ryan Armstrong <notifications@github.com>
wrote:

> In configure.ac
> <https://github.com/cavaliercoder/libzbxpgsql/pull/55#discussion_r76525600>
> :
>
> > @@ -1,4 +1,4 @@
> > -AC_INIT([libzbxpgsql], [1.1.0], [ryan@cavaliercoder.com])
> > +AC_INIT([libzbxpgsql], [1.1.1], [ryan@cavaliercoder.com])
>
> 1.1.0 hasn't been released yet, so this can stay as 1.1.0. I intend to
> include your changes in 1.1.0.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cavaliercoder/libzbxpgsql/pull/55/files/79848f2d016dd66c192d4d15c8fc91aae7a1d909#r76525600>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOBRoPylm-WaV8g2pcg4plt1Mq4aJx3-ks5qkQKsgaJpZM4Ju1Cu>
> .
>